client/src/pages/Bookings/BookingsTemplate.tsx

```diff
@@ -0,0 +1,136 @@
+  status: string;
+  id: number;
+}
+
+interface Props {
+  bookings: any;
+  sectionName: string;
+}
+
+export default function ManageBookings({ bookings, sectionName }: Props): JSX.Element {
+  const pageStyles = makeStyles((theme: Theme) =>
+    createStyles({
+      paper: {
+        padding: theme.spacing(2),
+        margin: 'auto',
+        maxWidth: 500,
+      },
+      image: {
+        width: 128,
+        height: 128,
+      },
+      img: {
+        margin: 'auto',
+        display: 'block',
+        maxWidth: '100%',
+        maxHeight: '100%',
+      },
+      text: {
+        textAlign: 'left',
+      },
+      sectionHr: {
+        fontWeight: 900,
+        fontSize: '9px',
+        color: 'black',
+        marginTop: '8px',
+      },
+      sectionWrapper: {
+        marginBottom: '-10px',
+      },
+      BookingSubHr: {
+        fontWeight: 600,
+        fontSize: '12px',
+        color: 'black',
+        margin: '10px 0px',
+      },
+      bookingNameHr: {
+        fontWeight: 900,
+        fontSize: '13px',
+      },
+      nameWrapper: {
+        display: 'flex',
+        marginBottom: '15px',
+      },
+      tempImg: {
+        borderRadius: '50%',
+        width: '20px',
+        marginRight: '10px',
+        backgroundColor: deepOrange[500],
+      },
+      paperOutlineWrap: {
+        marginBottom: '7px',
+        padding: '5px 10px',
+      },
+      iconWrapper: {
+        display: 'flex',
+        justifyContent: 'flex-end',
+      },
+      statusWrapper: {
+        display: 'flex',
+        alignItems: 'center',
+        fontSize: '8px',
+        color: 'rgb(209,209,209)',
+        fontWeigth: '900',
+      },
+      statusTxt: {
+        fontSize: '10px',
+        color: 'rgb(209,209,209)',
+        fontWeight: 900,
+      },
+    }),
+  );
+  const pageClasses = pageStyles();
+
+  return (
+    <Grid container spacing={2}>
+      <Grid item xs={12} sm container>
+        <Grid item xs container direction="column" spacing={2}>
+          <Grid item xs className={pageClasses.sectionWrapper}>
+            <Typography variant="body2" gutterBottom className={pageClasses.sectionHr}>
+              {sectionName}
+            </Typography>
+          </Grid>
+          <Grid item>
+            {bookings.map((booking: Booking) => (
+              <Paper variant="outlined" className={pageClasses.paperOutlineWrap} key={bookings.id}>
+                <Grid item container xs={12} sm>
+                  <Grid item xs={9}>
+                    <Typography gutterBottom variant="subtitle1" className={pageClasses.BookingSubHr}>
+                      {booking.date}
+                    </Typography>
+                    <Typography variant="body2" className={pageClasses.nameWrapper}>
+                      {/* TEMP IMG tags */}
+                      {/* <Avatar alt="Remy Sharp" src="../../Images/68f55f7799df6c8078a874cfe0a61a5e6e9e1687.png" /> */}
+                      <div className={pageClasses.tempImg}></div>
```

Comment:
You can use MUI `Box` components instead of divs

Reply:
done